### PR TITLE
Adding PyCO2SYS

### PR DESCRIPTION
**Pull request by mvdh7**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mvdh7** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/PyCO2SYS/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pyco2sys
```
[the remark this answers is not in the document]
Comment (mvdh7) [reply recipes/PyCO2SYS/meta.yaml:20]: Not sure, I haven't changed this from what was generated by grayskull

**Comment by dopplershift** [comment]
The error I saw indicated a need to add Pandas to the runtime dependencies.

**Comment by mvdh7** [comment]
> The error I saw indicated a need to add Pandas to the ~runtime~ host 🤨 dependencies.

Thanks, advice much appreciated - my first time doing this!

**Comment by dopplershift** [comment]
Here's the current problem:
```
 Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "/home/conda/staged-recipes/build_artifacts/pyco2sys_1675271483808/work/setup.py", line 6, in <module>
      with open("requirements.txt", "r") as f:
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
  error: subprocess-exited-with-error
```

Your `setup.py` needs to open `requirements.txt` in order to set the install requirements, but `requirements.txt` isn't being included in the source distribution you've uploaded to PyPI. (This is also one reason why I'm -1000 on `setup.py` reading `requirements.txt` files.)

**Pull request closed without merging**